.github/workflows/sgx-report.yml

```
@@ -0,0 +1,58 @@
+name: Create New SGX Report
+on: push
+jobs:
+  build_and_generate_report:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./notary-server
+    name: install Gramine
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+      - name: add gramine key
+        run: |
+               sudo curl -fsSLo /usr/share/keyrings/gramine-keyring.gpg https://packages.gramineproject.io/gramine-keyring.gpg
+               echo "deb [arch=amd64 signed-by=/usr/share/keyrings/gramine-keyring.gpg] https://packages.gramineproject.io/ $(lsb_release -sc) main" |  sudo tee /etc/apt/sources.list.d/gramine.list
+      - uses: awalsh128/cache-apt-pkgs-action@latest
+        with:
+          packages: rustc cargo gramine cmake clang gramine
```

duplicate, cargo includes rustc
```suggestion
          packages: cargo gramine cmake clang
```